### PR TITLE
Defficency Engineering Changes

### PR DESCRIPTION
**Pull request by DorainGrey**
Changelog:

- ~~Lobby Windows are now Shocked to prevent easy break ins.~~ Removed
- 1 box of metal foam grenades have now been added to atmos and engineering lobby.
- Security requests Console added to security desk as per https://github.com/vgstation-coders/vgstation13/issues/17788
- Two Oxygen jetpacks added to engineering hardsuit spaces. 
- Fixed Labeling of Secondary Engineering storage from Security to Engineering
- Toxins storage now has reinforced walls
- Moved Engineering Delivery up to the main lobby
- Added the parts need to rebuild telecomms into the tcomms storage/cooling room.
- Shard Engine has been redesigned, it is now identical to box/roid where you need to setup cooling loop externally, internally and add tanks.
- Singularity has been designed to be the same as synergy, lots of work but achievable.
- Also fixes this https://github.com/vgstation-coders/vgstation13/issues/31999
- Also added Disk Duplicator & disk box in library per https://github.com/vgstation-coders/vgstation13/issues/31106

image
image

First Mapping PR but I tested everything locally and it compiled/worked so should be good.

### CODE REVIEW

**Review by Jack7D1 — CHANGES_REQUESTED** [review]
image
image

Why'd you plug up the MULE door? I mean they're not used anyways, but still. Also if doing this make sure to get rid of the waypoint and to fix the scrubbers pipe below.

**Comment by gurfan** [comment]
fortressification

**Comment by DorainGrey** [comment]
There's already a mule door in the lobby and there was no point in leaving it behind a door.
I fixed the pipes & removed beacon.

**Comment by Kurfursten** [comment]
pretty sure that other mule door was for atmos, and this one is for engineering

check the waypoints in map

**Comment by Eneocho** [comment]
>Lobby Windows are now Shocked to prevent easy break ins.

Gay and lame. Engi is not command/sec.

>1 box of metal foam grenades have now been added to atmos and engineering lobby.

Engineers can get metal foam grenades from engivends and some extra grenades if hacked. I don't see the point of adding the box.

>Oxygen jetpacks added to engineering hardsuit spaces.

Why would engineering need o2 jetpacks? They're not planning on going space exploring. And if they are they can get em from EVA.

**Comment by adacovsk** [comment]
remove jetpacks and I'll updoot

**Comment by DorainGrey** [comment]
>  Lobby Windows are now Shocked to prevent easy break ins.

Removed

>  Engineers can get metal foam grenades from engivends and some extra grenades if hacked. I don't see the point of adding the box.

They can get 1 from a hacked engi vend which after used is unobtainable, hence why they are available on roid in a box, which is a good idea. Why gatekeep useful tools?

>  Oxygen jetpacks added to engineering hardsuit spaces.

There's two nitrogeon ones in the Engineering section.  I don't see why the actual engineering hardsuit are in the department can't have them either. Jetpacks are quite useful for working outside the station, like fixing breaches, setting up big plasma cooling loops and the like. I can change them to nitrogen if that's your issue.

**Comment by DorainGrey** [comment]
Updated based on Feedback:

- Removed Grille Shocking
- Moved Engineering Delivery up to the main lobby, re-added beacon.
- Redesigned the Singlo area to make it more viable to setup as a main engine.
- Added the parts need to rebuild telecomms into the tcomms storage/cooling room.

image

**Comment by DorainGrey** [comment]
Also added Disk Duplicator & disk box in library per https://github.com/vgstation-coders/vgstation13/issues/31106

File: maps/defficiency.dmm [hunk 1]
```diff
@@ -271502,17 +272997,17 @@ aaa
...
-aaa
-aaa
-aaa
+bxy
+aDV
+aHB
+bxu
+hSQ
+hSQ
+hSQ
```
Comment (Kurfursten) [inline maps/defficiency.dmm]: > three right particle emitters

You've made it impossible to set up the singularity by deleting the center and left emitters

**Comment by Eneocho** [comment]
> They can get 1 from a hacked engi vend which after used is unobtainable, hence why they are available on roid in a box, which is a good idea. Why gatekeep useful tools?

They get another for a coin, but your point does make sense. Still, I rarely see engineers use 'em.

File: maps/defficiency.dmm (hunk 1 above)
Comment (DorainGrey) [reply maps/defficiency.dmm]: Fixed, this and tested setup engine to make sure it works.

**Comment by DorainGrey** [comment]
- Fixed PA emitters
- Removed duplicate apc & air alarm
- Moved PA and grilles back one, to have easy access to rad collectors
- Reduced jetpacks to 2 in line with what other departments are issued with.
- Added first aid kit as there was none.
- Added an extinguisher in supermatter room.

**Comment by DorainGrey** [comment]
Hey if you are going to downvote this PR can you please explain why so I can fix it?
The supermatter is desperately in need of an upgrade on the map, anyone who plays engineering on that map will tell you so, when built as intended it delaminates within 5 minutes. Which means you are required to retrofit the engine quite a bit to make it safe. It doesn't even produce much power due to a lack of rad collectors and no spares available in storage.
The only alternative is either RUST or setting up an AME which doesn't fully meet the power requirements of the station.

**Comment by Kurfursten** [comment]
> when built as intended it delaminates within 5 minutes

This is the SME on all maps. You always need to change out the pipes, it's part of being an engineer

> doesn't produce much power

Good engineers only need 1 rad collector to power a station

You are probably wondering how engineers got by for 8 years on this map, because this is the original SME chamber from 2014! I remember when Duny added Deff originally, it was in his vision to not have a singulo on this map, as it would make the experience different from Box and Meta, encouraging engineers to become less reliant on one engine.

**Comment by WindowsErrors** [comment]
i just suck it up and wire the solars on deff

**Comment by Eneocho** [comment]
Just make the shard like a bigboye and power the station with a few 1013kpa room temp plasma.

**Comment by DorainGrey** [comment]
> This is the SME on all maps. You always need to change out the pipes, it's part of being an engineer

There's a difference between completing the engine with piping like on all other stations in rotation and having to replace most of the piping, undo the vents and add scrubbers like you do on deff. My vision of engineering is experimentation to make cool and interesting engines yourself, not re-plumb the engine that's deliberately worse than the rest of the maps. This design gives you an easy base to work on so you can try engineer with multiple shards, hot oxygen shards not just repeat the same steps from memory or the wiki every time....

This literally changes it to be the same as our other stations. You still need to add a cooling loop inside chamber, fill tanks, set scrubbers add a vent to space. I can remove the cooling loop existing in space and it'd be identical to box setup.

> doesn't produce much power

Sure, I cool plasma nearly every round but it really isn't a round start task. It's frustrating fixing the engine more than normal and then producing less power. Then when you want to do something actually interesting like aim for gigawatts having to order supplies from cargo to do so.

> You are probably wondering how engineers got by for 8 years on this map, because this is the original SME chamber from 2014! I remember when Duny added Deff originally, it was in his vision to not have a singulo on this map, as it would make the experience different from Box and Meta, encouraging engineers to become less reliant on one engine.

I totally agree with being more reliant on the supermatter as it's a better and more interesting engine. This is why the singularity is deliberately left incomplete in this PR, to encourage using the supermatter and experimenting. I simply made it a bit more viable to complete the engine if you really are dedicated to doing so for whatever reason. Unlike the existing framework which would require building a room to store the PA in and various other changes. This is similar to Roid & Synergy station.
I'll take it out if it's a big deal, I just thought it was a decent use for some dead space.

**Comment by pedr69** [comment]
>shocking grilles to make it harder to break in

lol. lmao. shocked grilles don't do shit except kill innocent bystanders after a bomb or a greyshit breaks the window guarding them
>redesigning the SM

imo it's fine; only thing i'd change is have one of the multiple loop filters go into a space vent instead of them all going into separate cans
i am in favor of adding metal foam grenades

**Comment by west3436** [comment]
>Hey if you are going to downvote this PR can you please explain why so I can fix it?

- Stop trying to make deff singularity happen
- Roundstart functional SME with no modifications is silly and even further removes the purpose of having Engineers on the station
- I liked the singularity storage room having access from maint so illicit activities could be done in there, mainly cult stuff
- The MULE delivery door into atmos was always a laff to use to break into Engineering, but I can understand why that'd be mapped out. Regardless, double-thick reinforced walls there reeks of grudgecode lmao
- Shocked grilles is very unnecessary for Engineering

**Comment by DorainGrey** [comment]
>     * Stop trying to make deff singularity happen
I'm not trying to make it happen, I hate the singularity engine, but since the map has all the equipment and a spot for one to be setup. I thought I'd at least make it viable to setup, otherwise we may as well delete it all which I would also be open to that if you want?
I just think synergy & roids half complete setups are nice and present a challenge, so I wanted to match that.

>     * Roundstart functional SME with no modifications is silly and even further removes the purpose of having Engineers on the station
It's not without modifications, you need setup a cooling loop inside the chamber, anchor the rad collectors and fit more emitters. That all takes longer than normal.
I'm going to change it to be exactly like box now, so there can be no more complaints about it.

>     * I liked the singularity storage room having access from maint so illicit activities could be done in there, mainly cult stuff
It's deff, there's dozens of free rooms to do this in? There's literally a construction area 10 tiles away which would be great for cult?

>     * The MULE delivery door into atmos was always a laff to use to break into Engineering, but I can understand why that'd be mapped out. Regardless, double-thick reinforced walls there reeks of grudgecode lmao
Double reinforced walls where? All the changes are contained in the original PR description, anything posted in comments is no longer relevant.

>     * Shocked grilles is very unnecessary for Engineering
These have been deleted long ago, see pull request description.

Thank you for not silent down-voting though, the emojiocracy is frustrating.

**Comment by DorainGrey** [comment]
- Singularity Redesigned to be same/more effort as Synergy/Roid, all gear is not set up. All that's provided is the room and the central catwalk.

- SME chamber redesigned to be identical to box stations in terms of effort. At a minimum you need to put in tanks, complete space cooling loop and complete chamber loop.

See PR Description for updated images.

**Comment by pedr69** [comment]
i dont like the new SM too much, as the only time you can make the SM "as intended", without pulling out the vents and replacing with HE inside the chamber is on deff
deff, roid, and synergy setups are great imo, they're different so you need to think how to make the pipes work, instead of just spending 5 minutes grunga piping
i'd like the different stations' setups to be different but almost functional to encourage engineers to complete the SM instead of tearing half the old one out and rebuilding with a functional one

you do have more playtime in engineering than i do so i guess if you think it sucks you know why better than i do
you have my updoot

**Comment by DamianX** [comment]
I share the mixed feelings others have regarding the singularity being more easily available, but everything else seems fine, so I'm thinking I'll merge this once the checks finish.

**Pull request merged**